docs: restructure and add more content (#2232)
caos/zitadel: docs/docs/guides/customization/branding.md

---

> title: Branding

Reviewer: ```suggestion
title: Brand Customization
```

---

> ZITADEL offers various customisation options for your branding. 

Reviewer: ```suggestion
ZITADEL offers various customization options for your projects brand design. 
```

Head over to the Private Labeling Policy on your Organization Page.

## How it works

> You are able to customize a light and a dark mode.

Reviewer: ```suggestion
You are able to customize the light and a dark mode separately.
```

Upload your logo for the chosen theme, as soon as it is uploaded the preview on the right side of the screen should show it.

### Colors
In the next part you can configure your colors. 
Background colour is self-explanatory, the primary color will be used for buttons, links and some highlights. 
The warn color is used for all the error messages and warnings and the font colour for texts. 

### Font
Last step to apply to your branding is the font upload. 
The best way is to upload a ttf file after a successful upload you will see it in the font part, but not in the preview.

### Advanced Settings
In the advanced behavior you can choose if the loginname suffix (domain e.g road.runner@acme.caos.ch) should be shown in the loginname screen or not and if the “ZITADEL watermark” should be hidden.

## Trigger the private labeling for the login
If you like to trigger your settings for your applications you have different possibilities.

### 1. Primary Domain Scope
Send a primary domain scope with your authorization request to trigger your organization.
The primary domain scope will restrict the login to your organization, so only users of your own organization will be able to login.

> ### 2. Setting on your Project
> Set the private labeling setting on your project to define which branding should trigger.
> 
> 
> ## Reset to default
> If you don't like your customization anymore click the "reset policy" button.
> All your settings will be removed and the default settings of the system will trigger.

Reviewer: ```suggestion
### 2. Setting on your Project
Set the private labeling setting on your project to define which branding should trigger.

## Reset to default
If you don't like your customization anymore click the "reset policy" button.
All your settings will be removed and the default settings of the system will trigger.
```
